### PR TITLE
Issue #1549 by Inlead: Implement customerror module.

### PR DESCRIPTION
**Pull request by inlead**
http://platform.dandigbib.org/issues/1549

### CODE REVIEW

File: ding2.install [hunk 1]
```diff
@@ -327,3 +327,40 @@ function ding2_update_7014() {
...
+      'format' => 'ding_wysiwyg',
+    );
+
+    variable_set('customerror_404_title', 'Siden blev ikke fundet');
+    variable_set('customerror_404', $content_404);
+    variable_set('site_404', 'customerror/404');
+
+    // Set the 403 for authenticated users.
```
Comment (guddo) [inline ding2.install]: I think you should group the two 403's anv move this above the L347.

**Comment by JrgenGNielsen** [comment]
reviewed and accepted

**Comment by GitCop** [comment]
Please ensure that your commit messages comply with our contribution guidelines.

 * Commit: 1f572896aac177f6c2add4f8df3e3880be89afe5
  * The subject should be kept around 50 characters long. The subject must not be more than 69 characters long. Strive for about 50 characters.

 * Commit: 45b8d3c31cd4e99018cd482e28d0dec7f00c04cf
  * The subject should be kept around 50 characters long. The subject must not be more than 69 characters long. Strive for about 50 characters.

 * Commit: f1d6fdbe36de6500cbe40dc0ca4d836ea6b679e6
  * The subject should be kept around 50 characters long. The subject must not be more than 69 characters long. Strive for about 50 characters.

Guidelines are available at https://github.com/ding2/ding2/blob/master/CONTRIBUTING.md.

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
Please ensure that your commit messages comply with our contribution guidelines.

 * Commit: 1f572896aac177f6c2add4f8df3e3880be89afe5
  * The subject should be kept around 50 characters long. The subject must not be more than 69 characters long. Strive for about 50 characters.

 * Commit: 45b8d3c31cd4e99018cd482e28d0dec7f00c04cf
  * The subject should be kept around 50 characters long. The subject must not be more than 69 characters long. Strive for about 50 characters.

 * Commit: f1d6fdbe36de6500cbe40dc0ca4d836ea6b679e6
  * The subject should be kept around 50 characters long. The subject must not be more than 69 characters long. Strive for about 50 characters.

Guidelines are available at https://github.com/ding2/ding2/blob/master/CONTRIBUTING.md.

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
Please ensure that your commit messages comply with our contribution guidelines.

 * Commit: 952afab3c283e9149d2a54a6fc282a4426daccc6
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

Guidelines are available at https://github.com/ding2/ding2/blob/master/CONTRIBUTING.md.

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
Please ensure that your commit messages comply with our contribution guidelines.

 * Commit: 952afab3c283e9149d2a54a6fc282a4426daccc6
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

Guidelines are available at https://github.com/ding2/ding2/blob/master/CONTRIBUTING.md.

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
Please ensure that your commit messages comply with our contribution guidelines.

 * Commit: 952afab3c283e9149d2a54a6fc282a4426daccc6
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

Guidelines are available at https://github.com/ding2/ding2/blob/master/CONTRIBUTING.md.

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
Please ensure that your commit messages comply with our contribution guidelines.

 * Commit: 952afab3c283e9149d2a54a6fc282a4426daccc6
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

Guidelines are available at https://github.com/ding2/ding2/blob/master/CONTRIBUTING.md.

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
Please ensure that your commit messages comply with our contribution guidelines.

 * Commit: 952afab3c283e9149d2a54a6fc282a4426daccc6
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

Guidelines are available at https://github.com/ding2/ding2/blob/master/CONTRIBUTING.md.

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
Please ensure that your commit messages comply with our contribution guidelines.

 * Commit: 952afab3c283e9149d2a54a6fc282a4426daccc6
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

Guidelines are available at https://github.com/ding2/ding2/blob/master/CONTRIBUTING.md.

---
This message was auto-generated by https://gitcop.com

**Comment by guddo** [comment]
Moved to https://github.com/ding2/ding2/pull/478.

**Comment by inlead** [comment]
Moved to https://github.com/ding2/ding2/pull/478.

**Pull request closed without merging**